python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -471,7 +499,7 @@ class _ActorPool:
     actors when the operator is done submitting work to the pool.
     """
 
-    def __init__(self, max_tasks_in_flight: int = float("inf")):
+    def __init__(self, max_tasks_in_flight: int = 4):
```

Comment:
is this the same value as `DEFAULT_MAX_TASKS_IN_FLIGHT`? if so, any way we can use the same constant?

Reply:
Done.